### DISCUSSION TITLE
PyART Cookbooks

### DISCUSSION THREAD

**Discussion by mgrover1** [ARM-DOE/pyart#1065] [Ideas]
We recently had a design review regarding "PyART Data Cookbooks", which would be in collaboration with ARM/DOE/Project Pythia. 

**Here is [a link to the cookbook](https://projectpythiatutorials.github.io/radar-cookbook/landing-page.html)**

Here is a link to [the presentation](https://docs.google.com/presentation/d/1CBa9K6ozvJyQuCZMSRYHa_hvp1EJqpYBIiaFPCdn58w/edit?usp=sharing) summarizing the proposed work

This is a graphic of 2 potential cookbooks - focusing on shared content, in addition to encapsulating end-to-end workflows.
<img width="1334" alt="Screen Shot 2022-03-08 at 9 31 42 AM" src="https://user-images.githubusercontent.com/26660300/157698553-e023cc49-2c20-47a8-8a60-d22cf693b60f.png">

What are people's thoughts? Opening this for discussion!

**Comment by mgrover1** [comment]
As a first step, here is a list of the **existing** PyART examples/resources

## PyART Docs
* [Examples on documentation - written as `.rst` files](https://arm-doe.github.io/pyart/examples/index.html)
* [Notebooks on documentation - written as `.ipynb` files](https://arm-doe.github.io/pyart/notebooks/basic_ingest_using_test_radar_object.html)

## Tutorials
* [PyART Training](https://github.com/ARM-Development/PyART-Training)
* [ARM-HPC Tutorial](https://github.com/ARM-DOE/armhpc-tutorial)
* [Open Source Radar Tutorial - Japan (2019)](https://github.com/openradar/AMS-Open-Source-Radar-2019)
* [Open Source Radar Tutorial - Australia (2018)](https://github.com/openradar/aus-open-radar-science-school)
* [European Radar Conference - Netherlands (2018)](https://github.com/openradar/erad2018/tree/master/pyart)
* [AMS Radar Data in the Cloud (2017)](https://github.com/openradar/AMS_radar_in_the_cloud)
* [AMS Open Radar Short Course (2017)](https://github.com/openradar/AMS-Short-Course-on-Open-Source-Radar-Software)
* [PyART Short Course (2015)](https://github.com/openradar/pyart_short_course)

## General Notebooks
* [ARM-notebooks](https://github.com/ARM-Development/ARM-Notebooks)

**Comment by scollis** [comment]
@mgrover1 can you put some links here to the latest cookbooks?

**Comment by mgrover1** [comment]
Hi ya'll - here is an update on Radar Cookbooks!

We have put together a few basic foundational notebooks, in addition to a workflow working with NEXRAD data! 

https://projectpythiatutorials.github.io/radar-cookbook/landing-page.html

Feel free to check out/leave feedback!

**Comment by mgrover1** [comment]
An update for those interested - we now have a "complete" radar cookbook, including an overview of the cookbook, foundational materials, and an example looking a supercell near Moore, Oklahoma. 

Here is the link to the book! https://projectpythiatutorials.github.io/radar-cookbook/landing-page.html
